ts/services/DashboardService.ts

```diff
@@ -46,12 +47,27 @@ class DashboardService {
         logger.info(response.dashboard.name);
       }
       if (config.format === 'json') {
+        // remove some internal keys
+        this.scrubDashboardKeys(response.dashboard);
         logger.info(JSON.stringify(response, null, 2));
       }
     } catch (e) {
       logger.error('There was an error getting the dashboard: ' + e);
     }
   }
+
+  private scrubDashboardKeys(dashboard) {
```

Comment:
This also feels like a great place to start adding some real tests.

Reply:
😭 I know